Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLReduction.py

```diff
@@ -589,6 +607,9 @@ def _finalize(self, ws, process):
         ReplaceSpecialValues(InputWorkspace=ws, OutputWorkspace=ws, NaNValue=0,
                              NaNError=0, InfinityValue=0, InfinityError=0)
         mtd[ws].getRun().addProperty('ProcessedAs', process, True)
+        mtd[ws].getRun().addProperty('numor_list', self._return_numors(self.getPropertyValue('Run')), True)
```

Comment:
This seems to output the full path including directory of the file.  Do you want this or just the filename? I'm fine with either but thought I'd double check

Reply:
`self.getPropertyValue('Run')` does naturally return full paths, and then `_return_numors `method removes the full path and returns only the file names and their extension. The header then contains only file names, binary operations, and file extensions.

Reply:
I think the code to strip the file name from the full path only works on linux because it relies on the forward slash character. Perhaps better to use os.path.basename(entry)?